all/CMakeLists.txt

```diff
@@ -1,13 +1,14 @@
-cmake_minimum_required(VERSION 3.8)
+cmake_minimum_required(VERSION 3.14...3.20)
 
 project(exampleAll NONE)
 
-set(example_INCLUDE_WITHOUT_SYSTEM YES CACHE INTERNAL "")
+#XXX set(example_INCLUDE_WITHOUT_SYSTEM YES CACHE INTERNAL "")
 
 add_subdirectory(.. sub/root)
 
 # The above add_subdirectory call creates the necessary target(s), add a stub
 file(WRITE "${PROJECT_BINARY_DIR}/configs/exampleConfig.cmake" "")
 set(example_DIR "${PROJECT_BINARY_DIR}/configs" CACHE INTERNAL "")
 
+enable_testing()
```

Comment:
It's not a superbuild CML's responsibility to muck with testing. It's already enabled as necessary in the test's project CML.

Reply:
you don't understand what I does here!

I test the installed `cmake config package` 
this is now the `cmake master project `and need this!

Reply:
Please speak in a coherent way and don't resolve conversations. I will resolve them when I consider them resolved.